.github/pull_request_template.md

```diff
@@ -0,0 +1,20 @@
+### Related Issue
+- Fixes # .
+
+### Proposed Changes
+- Change 1
+- Change 2
+
+### Additional Information
+- Any additional information about the change
+
+### Checklist
+- [ ] Tested
+- [ ] No Conflicts
+- [ ] Change In Code
+- [ ] Change In Documentation
+
+### Screenshots
```

Comment:
@Praneeth-rdy Just add "(Optional)" against the screenshot heading. And remove the remaining parts below it because certain PRs wont have any before-after SS or some may not even have any SS. We need to consider certain edge cases as well.